doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -474,6 +474,17 @@ arguments used for configuration.  Here is a table describing these emulated dev
           latter is ignored and the MAC address is set to the ``mac`` value.
           ``mac_seed`` will only be used when ``mac`` is not set.
 
+   * - ``virtio-gpu``
+     - Virtio GPU type device, parameters format is:
+       ``virtio-gpu,<geometry={widthxheight+x_off+y_off|fullscreen:screen_id},geometry={...}...]>``
+
+       * ``<geometry>`` specifies the mode of virtual display, windowed or fullscreen
+       * ``width`` specifies the width of the virtual display window.
+       * ``height`` specifies the height of the virtual display window.
+       * ``x_off`` specifies the x offfset of the virtual display window from origin point of the screen.
+       * ``y_off`` specifies the y offfset of the virtual display window from origin point of the screen.
+       * ``screen_id`` specifies the screen id when show virtual display as fullscreen.
```

Comment:
Is this `screen_id` the physical screen ID that will be used to display the virtual display as fullscreen?

How do you determine what the screen ID is? Is that related to the `DISPLAY` variable in any way?

Reply:
Yes, you are correct.

Reply:
Wouldn't it be better in this case to refer to call this `display_id` and talk about a "display" in our doc, so as to be consistent with the standard way of naming things?

Reply:
Please add more information to the documentation:
- For width and height, what is the unit of measurement?
- For x_off and y_off, what is the "origin point of the screen"? Will users know what it is?
- For screen_id, what is the ``DISPLAY`` variable, and where is it defined? cc: @gvancuts

Reply:
> Please add more information to the documentation:
> 
>     * For width and height, what is the unit of measurement?

Pixels

> 
>     * For x_off and y_off, what is the "origin point of the screen"? Will users know what it is?
> 
>     * For screen_id, what is the `DISPLAY` variable, and where is it defined? cc: @gvancuts

It's an environment variable, in a graphical interface, you can run `echo $DISPLAY` to see what it's set to (typically `:0`).

@psun3x , what is the format for the `screen_id` parameter?

Reply:
Just 0, 1, 2, ...

Reply:
> Just 0, 1, 2, ...

OK, we need a couple of things here:
* How do you determine which number maps to what monitor?
* Clearly indicate the format in the description (to avoid people like who may be tempted to try `:0` :-)

Reply:
We will use "monitor_id" instead of "screen_id". Till now we only support 1 monitor for fullscreen mode. We don't support multi-monitor and mapping yet.

Reply:
@psun3x At line 486, I think we are still missing information about how to determine the monitor_id value. Please add it to line 486, for example:

``monitor_id`` specifies the ID of the screen shown on the fullscreen virtual display, for example, 0. This value must match the value from <what?>.

@gvancuts @dbkinder Can you help with the wording above?

Reply:
The monitor_id is dynamic, not fixed because if a monitor plug-out. Till now, we only support 1 monitor, so, the current design don't have the algorithm how to generate monitor_id for multi-monitors. This format is for further design.

Reply:
If the ``fullscreen:<monitor_id>`` option isn't functional yet, I recommend removing it from this document. If the option does function, please add the valid value to the document. Users need to know what value(s) they should input.

Reply:
As my understanding, we

- Only support single virtual display per VM.
- Virtio-GPU support window mode and full screen mode
- For full screen mode, user has to indicate which physical monitor will be used.

So the fullscreen:<monitor_id> is functioanl already. Please correct me if my understanding is incorrect.

BTW, there's another question, if the monitor_id is only specific for full screen mode, then which monitor will be used for window mode?

Reply:
I just talked with Peng. Looks like we only support a single *physical* monitor. Whatever the monitor_id is, the window/fullscreen will always show in the primary monitor. Then let's remove the monitor_id first. And add it back once we support it.

Reply:
@ywan170 @psun3x Please update the doc per the above comment "Then let's remove the monitor_id first" (to ensure accuracy). See also dbkinder's comment below.